appveyor.yml

```diff
@@ -75,6 +67,11 @@ platform: x64
 skip_commits:
   files:
     - "*.md"
+    - .github/workflows/codecov-nightly.yml
+    - .github/workflows/*.sh
+    - .github/workflows/Dockerfile
+    - .github/workflows/LinuxImages.yml
+    - .github/workflows/testRE.R

```

Comment:
adding files which are irrelevant for the appveyour build (tests under Linux; code coverage) to the ignore list